build/build.proj

```diff
@@ -385,7 +385,10 @@
       <CoreInputTestAssemblies>@(TestAssemblyPath->WithMetadataValue("IsCore", "true"))</CoreInputTestAssemblies>
       <CoreInputTestAssembliesSpaced>$(CoreInputTestAssemblies.Replace(';', ' '))</CoreInputTestAssembliesSpaced>
 
-      <VsTestLogger>--TestAdapterPath:$(XunitXmlLoggerDirectory) --logger:"xunit;LogFileName=$(TestResultsFileName)-vsts.$(TestResultOutputFormat)" --logger:"console;verbosity=detailed" --settings:$(MSBuildThisFileDirectory)xunit.runsettings</VsTestLogger>
+      <VsTestLogger>--TestAdapterPath:$(XunitXmlLoggerDirectory)</VsTestLogger>
+      <VsTestLogger>$(VsTestLogger) --logger:"xunit;LogFileName=$(TestResultsFileName)-vsts.$(TestResultOutputFormat)"</VsTestLogger>
+      <VsTestLogger>$(VsTestLogger) --logger:"console;verbosity=detailed"</VsTestLogger>
+      <VsTestLogger>$(VsTestLogger) --settings:$(MSBuildThisFileDirectory)xunit-core.runsettings</VsTestLogger>
```

Comment:
This is opinionated (in other words my gut reaction is I prefer the old, long line syntax) and in a file otherwise not needed to be modified if you didn't rename the file.

Reply:
There's nothing wrong in expressing your opinions. I prefer splitting because it is easier to read in small screens (mobile devices). I will revert it back.